data-reconciliation-app/README.md

```diff
@@ -26,19 +33,66 @@ To get started run `cd data-reconciliation-app && make test` to run the applicat
 │   └── adrs            All Architecture decision records (ADR)
 │
 ├── governance
-│   └── constitution    CCF network constitution files
-│   └── nodes           CCF network nodes configs
-│   └── scripts         All governance scripts
-│   └── vote            Contains proposal voting acceptance and rejection logic
+│    └── constitution    CCF network constitution files
+│    └── nodes           CCF network nodes configs
+│    └── scripts         All governance scripts
+│    └── vote            Contains proposal voting acceptance and rejection logic
 │
 └── src                 Application source code
+│    └── endpoints      Application endpoints
+│    └── models         Domain models
+│    └── repositories   Data repositories
+│    └── services       Domain services
+│    └── utils          utility classes
 │
 └── test
-│    └── e2e-test       Application end to end tests
-│    └── unit-test      Application unit tests
+     └── e2e-test       Application end to end tests
+     └── unit-test      Application unit tests
+
+```
+
+## Getting Started
+
+To get started and run the application locally.
+
+```bash
+git clone https://github.com/microsoft/ccf-app-samples # Clone the samples repository
+code .                                                 # open samples repository in Visual studio code
+
+# In the VScode terminal window
+cd data-reconciliation-app                             # Navigate to reconciliation sample folder
+make build                                             # Run interactive end to end tests
 ```
 
-### Running Locally
+there several scenarios could be executed on the local environment
```

Comment:
```suggestion
There several scenarios could be executed on the local environment
```

Reply:
I feel like a lot of this information is repeated in the Testing section below? Shall we merge them together?

Reply:
I will move the testing part to the document of testing strategy and reference link to it.